alfresco5-hotfix-ALF-22094/src/main/amp/config/alfresco/module/alfresco5-hotfix-ALF-22094/alfresco-global.properties

```diff
@@ -1,2 +1,4 @@
-eu.xenit.alfresco.antiidle.cronexpression=* * * * * ?
-eu.xenit.alfresco.antiidle.cronstartdelay=0
+eu.xenit.alfresco.antiidle.cronexpression=0 0 * * * ?
```

Comment:
Tangential question: are dashes allowed in property names ?
`eu.xenit.alfresco.anti-idle.cronexpression` looks better :stuck_out_tongue:

Reply:
Good question, I will try it out